config/theme.go

```diff
@@ -108,6 +108,161 @@ func NewBaseTheme(preset Preset) *Theme {
 			},
 		}
 
+	// Special Preset for Protanopias
+	case PresetProtDark:
+		return &Theme{
+			Default: MustParseColor("white"),
+			Base: ThemeBase{
+				Key:       MustParseColorSlice("#feb927 / white"),
+				Info:      MustParseColor("white"),
+				Primary:   MustParseColor("#4860e6"),             // magenta
+				Secondary: MustParseColor("#2aabee"),             // cyan
+				Success:   MustParseColor("#6afd6a:bold"),        // bold green
+				Warning:   MustParseColor("#feb927:italic"),      // yellow
+				Danger:    MustParseColor("fg=white:bg=#c2270a"), // red background
+				Muted:     MustParseColor("#feb927"),             // yellow
```

Comment:
Maybe we want to set a different color for the muted instead for using the same as the warning color.

I set that for the default theme just to keep backwards compatibility for now, but I think we can have a separate color here for new themes.

Reply:
OK I set it to something yellowish light, that is green-ish when you're not color-blind... I guess we will adjust when we have some reviews...

Remember that what we see may be slightly strange, but this theme is not meant for normal use.